### ISSUE TITLE
group-exports aggregating modules should be valid

### ISSUE THREAD

**Issue by atikenny** [benmosher/eslint-plugin-import#1471]
I would like to make [aggregating modules](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) valid when using the `group-exports` rule.

**Use-case**
When aggregating modules in a single file the `group-exports` rule fails on because of multiple export declarations. This makes it invalid to re-export submodules from the root of a module.

**Steps to reproduce**
- create a file with the following exports
```js
export { default as module1 } from 'module-1'
export { default as module2 } from 'module-2'
```
- validate the file using the `group-exports` rule

**Expected results**
No linting error.

**Actual result**
Linting error.

**Comment by ljharb** [comment]
The specifiers being different should avoid the warning; if you had two of these statements from the same module, it should require you to consolidate them.
